source/_integrations/intellifire.markdown

```diff
@@ -67,3 +71,22 @@ The following is a description of the various diagnostic error sensors and what
  - **Offline** - Your appliance is currently offline.
  - **Pilot Flame** - Pilot Flame Error: Your appliance has been safely disabled. Please contact your dealer and report this issue.
  - **Soft Lock Out** - Sorry your appliance did not start. Try again by pressing Flame ON.
+
+
+### Troubleshooting
+
+The IFT module can suffer a variety of issues which will render it inoperable. Some of these have been confirmed by the manufacturer and some appear to be random. There are two paths to to try when attempting to reset the module:
+
+
+#### Issue a Soft Reset
+
+There is a debug web-interface available at: [iftapi.net/webaccess](http://iftapi.net/webaccess/login.html). From here you can login and then navigate to your individual fireplace unit and issue a **Soft Reset** as follows:
+
+![soft_reset.png](/images/integrations/intellifire/soft_reset.png)
+
+In some cases this will restore control to the fireplace, but if it doesn't you are best to try the following method.
```

Comment:
```suggestion
#### Issue a Soft Reset

There is a debug web interface available at [iftapi.net/webaccess](http://iftapi.net/webaccess/login.html). From here you can log in and then navigate to your individual fireplace unit and issue a **Soft Reset** as follows:

![soft_reset.png](/images/integrations/intellifire/soft_reset.png)

In some cases, this will restore control to the fireplace, but if it doesn't you are best to try the following method.
```